src/components/network/config-tables.js

```diff
@@ -674,10 +681,21 @@ export const TABLES_DEFINITIONS = {
                 id: 'Name',
                 dataKey: 'name',
                 columnWidth: MEDIUM_COLUMN_WIDTH,
+                changeCmd: "equipment.setName('{}')\n",
+                editor: ({ equipment, ...props }) =>
```

Comment:
I think you can simplify to :
editor: NameField,

Reply:
yes, right. I will simplify as requested